lint/backend.py

```diff
@@ -119,6 +119,16 @@ def translate_lineno_and_column(errors, offset):
 def get_lint_regions(linters, view):
     syntax = util.get_syntax(view)
     for (linter, settings) in linters:
+        selectors = settings.get('selectors', False)
+        if selectors:
+            for selector in selectors:
+                regions = view.find_by_selector(selector)
+                if regions:
+                    yield linter, settings, [region for region in regions]
+                    return
+            return
```

Comment:
Pretty sure these two `return`s should be `break`s.

Reply:
I should try gotos, haha. 👍

Reply:
Second one is probably a continue